api/src/main/java/org/pmiops/workbench/api/DataBrowserController.java

```diff
@@ -0,0 +1,292 @@
+    }
+    /**
+     * This method searches concepts
+     *
+     * @param analysisId
+     * @param stratum1
+     * @param stratum2
+     * @param stratum3
+     * @param stratum4
+     * @param stratum5
+     * @return
+     */
+    @Override
+    public ResponseEntity<AnalysisResultListResponse> getAnalysisResults(
+            Long analysisId,
+            String stratum1,
+            String stratum2,
+            String stratum3,
+            String stratum4,
+            String stratum5) {
+
+        final List<AnalysisResult> resultList;
+
+        if ( (stratum1 != null && !stratum1.isEmpty()) && (stratum2 != null && !stratum2.isEmpty()))  {
+             resultList = analysisResultDao.findAnalysisResultsByAnalysisIdAndStratum1AndStratum2(analysisId, stratum1, stratum2);
+        }
+        else if ((stratum1 != null && !stratum1.isEmpty())) {
+            resultList = analysisResultDao.findAnalysisResultsByAnalysisIdAndStratum1(analysisId, stratum1);
+        }
+        else {
+            resultList = analysisResultDao.findAnalysisResultsByAnalysisId(analysisId);
+        }
+
+        AnalysisResultListResponse resp = new AnalysisResultListResponse();
+        resp.setItems(resultList.stream().map(TO_CLIENT_ANALYSIS_RESULT).collect(Collectors.toList()));
+
+
+        return ResponseEntity.ok(resp);
+    }
+
+    @Override
+    public ResponseEntity<org.pmiops.workbench.model.AnalysisResult> getParticipantCount() {
+        long aid =  1; // partipant count analysis id
+        final List<AnalysisResult> resultList  =  analysisResultDao.findAnalysisResultsByAnalysisId(aid);
+        AnalysisResultListResponse resp = new AnalysisResultListResponse();
+        resp.setItems(resultList.stream().map(TO_CLIENT_ANALYSIS_RESULT).collect(Collectors.toList()));
+        if ( resp.getItems().size() > 0 ) {
+            return ResponseEntity.ok(resp.getItems().get(0));
+        }
+        else {
+            // Todo handle error better
+            org.pmiops.workbench.model.AnalysisResult r = new org.pmiops.workbench.model.AnalysisResult();
+            r.setAnalysisId( (long) 1);
+            r.setCountValue((long) 0);
+            return ResponseEntity.ok(r);
+        }
+    }
+
+    @Override
+    public ResponseEntity<AnalysisResultListResponse> getConceptCount(String conceptId) {
+        long aid = 3000;
+        return this.getAnalysisResults(aid, conceptId, null, null , null, null);
+    }
+
+    @Override
+    public ResponseEntity<AnalysisResultListResponse> getConceptCountByGender(String conceptId) {
+        long aid = 3101;
+        return this.getAnalysisResults(aid, conceptId, null, null,null,null);
+    }
+
+    @Override
+    public ResponseEntity<AnalysisResultListResponse> getConceptCountByAge(String conceptId) {
+        long aid = 3102;
+        return this.getAnalysisResults(aid, conceptId, null, null , null, null);
+    }
+
+    @Override
+    public ResponseEntity<AnalysisListResponse> getAnalyses() {
```

Comment:
Why do we want to list all the analyses? Should we sort by name? Paginate? (How many are there going to be?)